lib/Service/TableTemplateService.php

```diff
@@ -469,8 +469,8 @@ private function makeVacationRequests(Table $table):void {
 				'title' => $this->l->t('Create Vacation Request'),
 				'emoji' => '️➕',
 				'columns' => json_encode([$columns['employee']->getId(), $columns['from']->getId(), $columns['to']->getId(), $columns['workingDays']->getId(), $columns['dateRequest']->getId()]),
-				'sort' => json_encode([["columnId" => $columns['from']->getId(), "mode" => "ASC"]]),
-				'filter' => json_encode([[["columnId" => $columns['employee']->getId(), "operator" => "is-equal", "value" => "@my-name"], ["columnId" => $columns['approved']->getId(), "operator" => "is-empty", "value" => ""]]]),
+				'sort' => json_encode([["columnId" => -5, "mode" => "ASC"]]),
```

Comment:
Constants not used in this file (also the changes below)

Reply:
Pushed another commit for two more places as well.